app/src/services/Help.js

```diff
@@ -40,14 +37,32 @@ class HelpService {
   getAllHelpForUser() {}
   getAllHelpForHelper() {}
 
-  async createHelp(title, categoryId, description) {
-    const createdHelpResponse = await api.post("/help", {
+  async createHelp(title, categoryId, description, accessToken, ownerId) {
+    const data = {
       title,
       categoryId,
       description,
-      ownerId: "5e8e4e19c2ebbc0026761416",
-    });
-    return createdHelpResponse;
+      ownerId,
+    };
+    const headers = {
+      Authorization: accessToken,
+    };
+    const createdHelpResponse = await api.post("/help", data, { headers });
+
+    return createdHelpResponse.data;
+  }
+
+  async chooseHelp(idHelp, idHelper, accessToken) {
```

Comment:
```suggestion
  async chooseHelp(idHelp, idHelper, accessToken) {
     try {
      const headers = {
        Authorization: accessToken,
      };

      const url = `/help/possibleHelpers/${idHelp}/${idHelper}`;
      await api.put(url, { headers });
    } catch (error) {
      console.log(error.response);
    }
  }
  }
```

Resolver os conflitos desse arquivo